aten/src/ATen/native/mkldnn/Conv.cpp

```diff
@@ -175,26 +179,87 @@ static inline at::MemoryFormat mkldnn_convolution_memory_format(int64_t dims, bo
    return memory_format;
 }
 
-void _mkldnn_convolution_out (
+#define ONEDNN_PARAM_TYPE_CONV 0
+#define ONEDNN_PARAM_TYPE_LINEAR 1
+bool mkldnn_delete_param(int64_t param, int64_t param_type) {
```

Comment:
The purpose of this function is to actually delete the param, instead of handle, so how about we update the function schema as `bool mkldnn_delete_param(int64_t param_handle, int64_t param_type)` ?